base/db/tests/Rcheck_reference.log

```diff
@@ -62,7 +62,8 @@ The Date field is over a month old.
 * checking package directory ... OK
 * checking for future file timestamps ... OK
 * checking ‘build’ directory ... OK
-* checking DESCRIPTION meta-information ... OK
+* checking DESCRIPTION meta-information ... NOTE
+License stub is invalid DCF.
```

Comment:
This note is newly added in R 4.4. the best way to fix it is to finish relicensing each package as BSD.